thread-rcu/Makefile

```diff
@@ -0,0 +1,19 @@
+CFLAGS = -Wall
+CFLAGS += -g
+CFLAGS += -std=c89
```

Comment:
In this repository, C99 is assumed. Linux kernel is now shifting to C99 as well.